speech/google/cloud/speech/sample.py

```diff
@@ -69,6 +71,15 @@ def __init__(self, content=None, source_uri=None,
             raise ValueError('Invalid encoding: %s' % (encoding,))
 
     @property
+    def chunk_size(self):
+        """Chunk size to send over GRPC. ~100ms
+
+        :rtype: int
+        :returns: Optimized chunk size.
+        """
+        return int(self.sample_rate / 10)
```

Comment:
What do you want this to do? The division will truncate and casting to int will truncate. Are you trying to round? Does it matter?